analyzers/src/SonarAnalyzer.CSharp/Rules/DoNotShiftByZeroOrIntSize.cs

```diff
@@ -128,7 +127,7 @@ protected override void Initialize(SonarAnalysisContext context)
                 return new Tuple<Shift, ExpressionSyntax>(Shift.Left, assignmentExpession.Right);
             }
 
-            if (assignmentExpession?.OperatorToken.IsKind(SyntaxKind.GreaterThanGreaterThanEqualsToken) ?? false)
+            if (assignmentExpession?.OperatorToken.IsAnyKind(SyntaxKind.GreaterThanGreaterThanEqualsToken, SyntaxKindEx.GreaterThanGreaterThanGreaterThanEqualsToken) ?? false)
             {
                 return new Tuple<Shift, ExpressionSyntax>(Shift.Right, assignmentExpession.Right);
             }
```

Comment:
NITPICK: In TryGetConstantValue the && operator is at the end of the line however, it should be at the beginning of the next one.